plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -43,8 +43,23 @@ import (
 	_ "k8s.io/kubernetes/pkg/apis/imagepolicy/install"
 )
 
-// PluginName indicates name of admission plugin.
-const PluginName = "ImagePolicyWebhook"
+const (
+	// PluginName indicates name of admission plugin.
+	PluginName = "ImagePolicyWebhook"
+
+	// AuditKeyPrefix is used as the prefix for all audit keys handled by
+	// this pluggin. Some well known suffixes are listed below.
+	AuditKeyPrefix = PluginName + ".image-policy.k8s.io/"
```

Comment:
are upper-case prefix names allowed in audit annotations? I didn't think they were

Reply:
@tallclair @sttts seems like audit annotations should allow the qualified keys metadata annotations does

Reply:
The spec says it should be the plugin name.  Should we just change the plugin name to all lower case?

Reply:
Yeah, lowercase is required. We should update the documentation to clarify.

BTW - here is the validation we use on the annotation key:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L37-L70

Reply:
that was for webhook-returned audit annotations... webhook admission plugin names are lowercase... I would lowercase the name for purposes of this annotation